rtp_to_hls/README.md

```diff
@@ -47,24 +43,102 @@ mix deps.get
 ```
 
 You may be asked to install `Hex` and then `rebar3`.
-In case of installation issues with Hex on Ubuntu, try updating the system packages first by entering the command:
+
+> In case of issues with compilation of membrane_h264_ffmpeg_plugin, enter:
+>
+> ```shell
+> mix deps.update bundlex
+> ```
+>
+> and then install pkg-config:
+>
+> ```shell
+> brew install pkg-config
+> ```
+
+Finally ,you can run the demo with:
```

Comment:
```suggestion
Finally, you can run the demo with:
```

Reply:
And I think there are more similar mistakes

Reply:
I will check all files once again and correct it